mut/index/SnootyManifest.py

```diff
@@ -142,7 +142,9 @@ def find_metadata(self):
         results = jsonpath_expr.find(self.tree)
         if results:
             results = results[0].value
-            if "robots" in results and (results["robots"] == "None" or "noindex" in results["robots"]):
+            if "robots" in results and (
+                results["robots"] == "None" or "noindex" in results["robots"]
```

Comment:
does case matter for the `"None"` comparison?

Reply:
this is change from black formatting and should not change logic here... that said, i dont think case matters here. None, as in from parser output, i believe